packages/opencensus-core/test/test-time-util.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import {getTimestampWithProcessHRTime} from '../src/common/time-util';
+
+describe('getTimestampWithProcessHRTime()', () => {
+  it('compare now with getTimestampWithProcessHRTime', () => {
```

Comment:
What would you think about using time mocking here and adding tests that cover the cases where the nanoseconds both do and don't overflow?

What you could do is move the initialization logic in `common/time-util.ts` above into a function say `setupHrtimeDrift` or similar, then you would add a new exported variable `export TEST_ONLY = {setupHrtimeDrift}` with a comment that is exported to enable testing the drift configuration (and of course you call the `setupHrtimeDrift` in the `time-util.ts` file). 

Then you import the `TEST_ONLY` symbol here in the test and call `setupHrtimeDrift` after you have mocked `process.hrtime` and `Date.now`. Then you redefine them again before calling `getTimestampWithProcessHRTime` for the two cases to test.

If this seems too involved, I'm open to leaving this as-is though.